### ISSUE TITLE
SSL issue

### ISSUE THREAD

**Issue by dubeychandan996** [swan-io/swan-partner-frontend#658]
dev server started

Banking: https://banking.swan.local:8080/
Onboarding Individual: https://onboarding.swan.local:8080/onboarding/individual/start
Onboarding Company: https://onboarding.swan.local:8080/onboarding/company/start

above things works fine ..
now i want custom subdomain of every service and when I try with reversepoxy of apach2 I am getting an error

Proxy Error
The proxy server could not handle the request
Reason: Error during SSL Handshake with remote server

i tried nginx too
NGINX 
Missing ALPN Protocol, expected h2 to be available.
If this is a HTTP request: The server was not configured with the allowHTTP1 option or a listener for the unknownProtocol event.

in short:

curl https://payment.swan.local:8080 --http1.1
curl: (35) error:0A000460:SSL routines::reason(1120)

**Comment by bloodyowl** [comment]
did you generate your wildcard certificate?

**Comment by dubeychandan996** [comment]
For these below i use self signed
Banking: https://banking.swan.local:8080/
Onboarding Individual: https://onboarding.swan.local:8080/onboarding/individual/start
Onboarding Company: https://onboarding.swan.local:8080/onboarding/company/start


and for subdomains that i am thinking to point i use let’s encrypt ssl
